.github/workflows/wiki.yml

```
@@ -10,6 +10,24 @@ on:
   gollum:
 
 jobs:
+  table_of_contents_check:
+    name: Wiki Check Table of Contents match Headers
```

```suggestion
    name: Check Wiki Table of Contents
```

You can add more clarifying details as a comment above it.

---

Done! I've made the name concise and moved the extra info as a comment above it.